orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/ExecutionMapper.kt

```diff
@@ -96,4 +102,31 @@ class ExecutionMapper(
           }
       )
   }
+
+  private fun convertPipelineRefTrigger(execution: PipelineExecution, context: DSLContext) {
+    val trigger = execution.trigger
+    if (trigger is PipelineRefTrigger) {
+      val parentExecution = context
+        .select(listOf(
+          field("id"),
+          field("body"),
+          field(name("partition"))
+        ))
+        .from(ExecutionType.PIPELINE.tableName)
+        .where(field("id").eq(trigger.parentExecutionId))
+        .fetchExecutions(mapper, 200, context)
+        .firstOrNull()
+
+      if (parentExecution == null) {
+        // If someone deletes the parent execution, we'll be unable to load the full, valid child pipeline. Rather than
+        // throw an exception, we'll continue to load the execution with [PipelineRefTrigger] and let downstream
+        // consumers throw exceptions if they need to. We don't want to throw here as it would break pipeline list
+        // operations, etc.
+        log.error("Attempted to load parent execution for '${execution.id}', but it no longer exists: ${trigger.parentExecutionId}")
```

Comment:
nit: I would make this a `warn` so it doesn't show up sentry, etc.
Also, it's not really an error here, as you state. It's an error at a time when someone downstream needs it